vendor/acquia/blt/config/build.yml

```diff
@@ -0,0 +1,275 @@
+# @todo Move to subkey of tests.
+behat:
+  config: ${repo.root}/tests/behat/local.yml
+  profile: local
+  # An array of paths with behat tests that should be executed.
+  paths:
+    # - ${docroot}/modules
+    # - ${docroot}/profiles
+    - ${repo.root}/tests/behat
+  tags: '~ajax&&~experimental&&~lightningextension'
+  extra: ''
+  validate: true
+  # May be chrome, selenium.
+  web-driver: chrome
+
+bin.path: vendor/bin
+
+blt:
+  update:
+    ignore-existing-file: ${blt.root}/scripts/blt/ignore-existing.txt
+  config-files:
+    project: ${repo.root}/blt/blt.yml
+    local: ${repo.root}/blt/local.blt.yml
+    example-local: ${repo.root}/blt/example.local.blt.yml
+    schema-version: ${repo.root}/blt/.schema_version
+  command-cache-dir: ${blt.root}/cache/commands
+
+composer:
+  bin: ${repo.root}/${bin.path}
+  extra: ''
+
+cm:
+  # Possible values: core-only, config-split, features, none.
+  strategy: config-split
+  core:
+    # The parent directory for configuration directories, relative to the docroot.
+    path: ../config
+    # The default config key to use for imports. This is the key used in Drupal's global $config_directories variable.
+    # E.g., $config_directories['sync']. It must have a corresponding key in cm.core.dirs. E.g., `cm.core.dirs.sync`.
+    key: sync
+    dirs:
+      # Corresponding value is defined in config.settings.php.
+      sync:
+        path: ${cm.core.path}/default
+    # Install site directly from existing config.
+    # This cannot be used if your install profile implements hook_install.
+    # Import our project config
+    install_from_config: true
```

Comment:
added this to import our project config

Reply:
@LindsayYoung Looking at what this does in the BLT code ... 👀 

https://github.com/acquia/blt/blob/46203b6c9c55ef142c9bd9150c7e6d2a14bfd5a0/src/Robo/Commands/Setup/DrupalCommand.php#L61